### DISCUSSION TITLE
Differentiate styling of epigraph, pull-quote and highlight elements

### DISCUSSION THREAD

**Discussion by tuukkamustonen** [pradyunsg/furo#258] [Ideas] [answered]
[Epigraph](https://docutils.sourceforge.io/docs/ref/rst/directives.html#epigraph), [highlights](https://docutils.sourceforge.io/docs/ref/rst/directives.html#highlights) and [pull-quote](https://docutils.sourceforge.io/docs/ref/rst/directives.html#pull-quote) look 100% alike now:

```rst
.. epigraph::

   No matter where you go, there you are.

   -- Buckaroo Banzai

.. highlights::

   This is a highlight.

.. pull-quote::

   Pull quote example.
```

![image](https://user-images.githubusercontent.com/94327/134633980-cfcdee4e-d137-4393-aa75-c6fb43c807e7.png)

Wouldn't it make sense to differentiate them a bit?

Using italic for `highlights` should be at least removed and perhaps that block should be, indeed, highlighted a bit?

---

All these generate `blockquote` HTML element but differing class names:

```html
<blockquote class="epigraph">...
<blockquote class="highlights">...
<blockquote class="pull-quote">...
```

...which is maybe a bit weird, because "highlights" definitively doesn't sound like quotation (so why does docutils render it as `blockquote`? Don't know).

Currently, the styling is done on `blockquote` element directly, so the same gets applied to all three.

**Comment by pradyunsg** [comment]
Got examples of how some other Sphinx themes present these?

**Reply by tuukkamustonen** [reply]
sphinx-book-theme [seems to have some opinion](https://sphinx-book-theme.readthedocs.io/en/latest/content-blocks.html) on `pull-quote` vs `epigraph`. Nutshell: Epigraph is larger, a pull-quote smaller.

insipid-sphinx-theme [seems to have similar convention](https://insipid-sphinx-theme.readthedocs.io/en/latest/showcase/quotes.html). It also implements `highlights` (without text being italic unlike with pull-quote/epigraph). Scroll to bottom of the page to see them.

sphinx-rtd-theme [only defines block quote](https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/demo.html#block-quotes) but none of the 3 directives.

It's a bit hard to find examples of these... They are probably not that often used elements, perhaps something one discovers only after years of writing docs (happened to me!).

(All these should be added to Furo's Kitchen Sink, too.)

**Comment by pradyunsg** [comment, marked as answer]
This was done at some point: https://pradyunsg.me/furo/kitchen-sink/blocks/

**Reply by tuukkamustonen** [reply]
Excellent. Thanks!
